developer_manual/scripts/theme-bootstrap.sh

```diff
@@ -12,31 +12,47 @@ function clone_example_theme
 {
   local APP_NAME="$1"
   local INSTALL_BASE_DIR="$2"
-  local DOWNLOAD_FILE=/tmp/master.zip
+  local MAINFILE=master.zip
+  local UNZIPDIR=/tmp
+  local MASTERNAME=theme-example-master
+  local DOWNLOAD_FILE=$UNZIPDIR/$MAINFILE
   local THEME_ARCHIVE_URL=https://github.com/owncloud/theme-example/archive/master.zip
 
+  # check if the app name already exists
   if  [ -d "$INSTALL_BASE_DIR/$APP_NAME" ]
   then
-    echo "The example theme ('$INSTALL_BASE_DIR/$APP_NAME') already exists."
+    echo "An app with name ('$INSTALL_BASE_DIR/$APP_NAME') already exists."
     echo "Please remove or rename it before running this script again."
     return 1
   fi;
 
-  echo "Cloning ownCloud example theme."
-  echo
+  # delete an existing downloaded zip file
+  if [ -e "$DOWNLOAD_FILE" ]
```

Comment:
I'm strongly for the original, shorter syntax here, as the longer syntax offers no added value.

Reply:
This is correct, but I wanted to have all queries looking the same for ease of reading for the less experienced 😄

Reply:
Point taken, but I don't see a lot of (if any) difference. Plus, I'm all for encouraging best practice in whatever language is being used.